.travis.yml

```diff
@@ -6,35 +6,32 @@ python:
 matrix:
   include:
 
-    - env: COMPILER_VERSION=3.6
-      compiler: clang
-      addons: &clang36
+    - env: CC_COMPILER="./test/wasm-install/wasm-install/bin/clang" CXX_COMPILER="./test/wasm-install/wasm-install/bin/clang++"
+      compiler: gcc
```

Comment:
We write gcc and use the gcc5 addon, so that it installs the same gcc5 toolchain in all builds in the matrix. This in particular ensures we install new-enough versions of libstdc++ and so forth, since what works for gcc5 also works for recent clang in terms of c++11+ support. Then, in the last build we actually use gcc5, and in the others, we override to use the waterfall clang.

Just writing clang here isn't enough, as it uses old clang which does not install what waterfall clang needs. We would need to both write clang and use the clang36 addon, but that's what LLVM broke... Alternatively we could find the specific packages we need instead of using gcc5, however (1) we use gcc5 in the last build anyhow, so this seems simpler, and (2) I have no idea how to find the right packages ;)